client: panic on DEX suspension.

1. Create account and get required confs.
2. Navigate to a market in browser.
3. Suspend that market using the admin console.
4. Attempt to reload the page using browser and log in.
<details closed>
<summary>client logs</summary>

```
2020-06-18 18:45:11.436 [DBG] CORE: new client core created
2020-06-18 18:45:11.436 [INF] CORE: started DEX client core
2020-06-18 18:45:11.437 [INF] CORE[dcr]: Setting up new DCR wallet at 127.0.0.1:19567 with TLS certificate "/home/joe/stuff/system/simnetconfs/dextest/dcr/alpha/rpc.cert".
2020-06-18 18:45:11.437 [INF] CORE: Loaded dcr wallet configuration. Account "", Deposit address = SsnJ6FLXfBWeBGF39jzrH8ug7AtquPdugoD
2020-06-18 18:45:11.437 [INF] CORE: successfully loaded 1 of 1 wallets
2020-06-18 18:45:11.448 [INF] RPC: RPC server listening on localhost:5757
2020-06-18 18:45:11.449 [INF] WEB: Web server listening on http://localhost:5758
2020-06-18 18:45:11.536 [INF] CORE: Connected to DEX server at 127.0.0.1:7232 and listening for messages.
2020-06-18 18:45:11.536 [DBG] CORE: connectDEX for 127.0.0.1:7232 completed, checking account...
2020-06-18 18:45:11.536 [DBG] CORE: dex connection to 127.0.0.1:7232 ready
2020-06-18 18:45:11.536 [INF] CORE: Successfully connected to 1 out of 1 DEX servers
2020-06-18 18:45:11.537 [INF] CORE: notify: |SUCCESS| (conn) DEX connected - DEX at 127.0.0.1:7232 has connected
12020-06-18 18:45:15.372 [DBG] WEB: New websocket client 127.0.0.1
2020-06-18 18:45:15.376 [TRC] WEB: Disconnected websocket client 127.0.0.1
2020-06-18 18:45:15.451 [DBG] WEB: New websocket client 127.0.0.1
2020-06-18 18:45:16.974 [DBG] CORE: authenticated connection to 127.0.0.1:7232
2020-06-18 18:45:17.006 [TRC] WEB: message of type 1 received for route loadmarket
2020-06-18 18:45:20.004 [TRC] CORE: notify: |DATA| (epoch) - Index: 159247352
2020-06-18 18:45:31.345 [TRC] WEB: Disconnected websocket client 127.0.0.1
2020-06-18 18:45:31.466 [DBG] WEB: New websocket client 127.0.0.1
2020-06-18 18:45:32.485 [TRC] WEB: message of type 1 received for route loadmarket
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9c5c36]

goroutine 183 [running]:
decred.org/dcrdex/client/order.(*bookSide).BestNOrders(0x0, 0x7fffffffffffffff, 0x0, 0x0, 0x0, 0x0)
        /home/joe/git/dcrdex/client/order/bookside.go:136 +0x66
decred.org/dcrdex/client/order.(*bookSide).orders(...)
        /home/joe/git/dcrdex/client/order/bookside.go:130
decred.org/dcrdex/client/order.(*OrderBook).Orders(0xc0005bf7c0, 0xc14a00, 0xc0007f6b58, 0x451f27, 0x0, 0xc0007f6b70, 0x3, 0x3, 0xc0005d8110, 0x7)
        /home/joe/git/dcrdex/client/order/orderbook.go:413 +0x42
decred.org/dcrdex/client/core.(*bookie).book(0xc0005bf7c0, 0xc000155ec0)
        /home/joe/git/dcrdex/client/core/bookie.go:139 +0x42
decred.org/dcrdex/client/core.(*Core).Sync(0xc0000cc900, 0xc0005d8100, 0xe, 0x2a, 0x0, 0x0, 0x0, 0x0)
        /home/joe/git/dcrdex/client/core/bookie.go:164 +0xb19
decred.org/dcrdex/client/webserver.wsLoadMarket(0xc000902900, 0xc0002eea80, 0xc0000c6640, 0xa)
        /home/joe/git/dcrdex/client/webserver/websocket.go:175 +0x1de
decred.org/dcrdex/client/webserver.(*WebServer).handleMessage(0xc000902900, 0xc0002eea80, 0xc0000c6640, 0xc0005c6210)
        /home/joe/git/dcrdex/client/webserver/websocket.go:132 +0x233
decred.org/dcrdex/client/webserver.(*WebServer).websocketHandler.func1(0xc0000c6640, 0x5e)
        /home/joe/git/dcrdex/client/webserver/websocket.go:78 +0x40
decred.org/dcrdex/dex/ws.(*WSLink).inHandler(0xc000922960, 0xf11e20, 0xc00002f140)
        /home/joe/git/dcrdex/dex/ws/wslink.go:236 +0x1d8
created by decred.org/dcrdex/dex/ws.(*WSLink).Connect
        /home/joe/git/dcrdex/dex/ws/wslink.go:169 +0x30f
[joe@hyrule dexc]$
```
</details>

<details closed>
<summary>server logs</summary>

 ```
2020-06-18 18:45:07.933 [TRC] COMM: Starting RPC server
2020-06-18 18:45:07.933 [INF] COMM: RPC server listening on 127.0.0.1:7232
2020-06-18 18:45:07.940 [INF] MAIN: The DEX is running. Hit CTRL+C to quit...
2020-06-18 18:45:07.940 [INF] ADMN: admin server listening on 127.0.0.1:6542
2020-06-18 18:45:11.535 [DBG] COMM: New websocket client 127.0.0.1
2020-06-18 18:45:11.535 [TRC] COMM: Starting websocket messaging with peer 127.0.0.1
2020-06-18 18:45:16.978 [DBG] AUTH: User 19faa20cc02d0286fb952d3e9cb2039f2693c550bce0dc0086366585491ccb2b connected from 127.0.0.1 with 0 pending requests and 0 pending responses/notifications.
2020-06-18 18:45:27.639 [INF] ADMN: server authenticated ip: 127.0.0.1:36126
2020-06-18 18:45:27.639 [INF] COMM: Broadcasting notification for route suspension to 1 clients...
2020-06-18 18:45:27.639 [TRC] COMM: Broadcast: "{\"type\":3,\"route\":\"suspension\",\"payload\":{\"marketid\":\"dcr_btc\",\"finalepoch\":159247352,\"suspendtime\":1592473530000,\"persistbook\":true}}"
2020-06-18 18:45:30.000 [INF] MKT: Book order feed closed for market "dcr_btc" after epoch 159247352, persist book = true.
2020-06-18 18:45:30.000 [INF] MKT: Book router terminating for market "dcr_btc"
2020-06-18 18:45:30.005 [DBG] MKT: epoch pump drained for market dcr_btc
2020-06-18 18:45:30.005 [INF] MKT: Market "dcr_btc" stopped.
2020-06-18 18:45:32.490 [ERR] COMM: Websocket receive error from peer 127.0.0.1: websocket: close 1006 (abnormal closure): unexpected EOF
2020-06-18 18:45:32.490 [TRC] COMM: Websocket output handler done for peer 127.0.0.1
2020-06-18 18:45:32.490 [INF] COMM: Shutting down link for 127.0.0.1 with 0 queued messages.
2020-06-18 18:45:32.490 [TRC] COMM: Disconnected websocket client 127.0.0.1
```
</details>

--- a/client/order/orderbook.go
+++ b/client/order/orderbook.go
@@ -62,6 +62,9 @@ type OrderBook struct {
 func NewOrderBook() *OrderBook {
 	ob := &OrderBook{
 		noteQueue:  make([]*cachedOrderNote, 0, defaultQueueCapacity),
+		orders:     make(map[order.OrderID]*Order),
+		buys:       NewBookSide(descending),
+		sells:      NewBookSide(ascending),
 		epochQueue: NewEpochQueue(),
 	}
 	return ob
@@ -181,6 +184,8 @@ func (ob *OrderBook) Sync(snapshot *msgjson.OrderBook) error {
 	ob.seqMtx.Unlock()
 
 	ob.marketID = snapshot.MarketID
+
+	// Clear all orders, if any.
 	ob.orders = make(map[order.OrderID]*Order)
 	ob.buys = NewBookSide(descending)
 	ob.sells = NewBookSide(ascending)

